.JuliaFormatter.toml

```diff
@@ -1 +1,2 @@
-style = "sciml"
\ No newline at end of file
+style = "sciml"
+annotate_untyped_fields_with_any = false
```

Comment:
isn't this already in the sciml format?

Reply:
It doesn't do that 😅

Reply:
https://github.com/domluna/JuliaFormatter.jl/blob/c72d049210a06e2a404df51998051f3dbb0a123b/src/styles/sciml/pretty.jl#L25

It seems okay to have it as true by default, but if we use something like ConcreteStructs (which IMO we should be using everywhere), then it becomes a problem.